packages/canary-bot/cloudbuild.yaml

```diff
@@ -48,3 +48,34 @@ steps:
       - "$_FUNCTION_REGION"
       - "$_REGION"
       - "$_DIRECTORY"
+
+  - name: gcr.io/cloud-builders/docker
+    id: "build-docker"
+    waitFor: ["-"]
+    dir: packages/canary-bot
+    args:
+      - "build"
+      - "-t"
+      - "gcr.io/$PROJECT_ID/canary-bot"
+      - "."
+
+  - name: gcr.io/cloud-builders/docker
+    id: "push-docker"
+    waitFor: ["build-docker"]
+    args: ["push", "gcr.io/$PROJECT_ID/canary-bot"]
+  
+  - name: gcr.io/cloud-builders/gcloud
+    id: "deploy-cloud-run"
+    waitFor: ["push-docker"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./scripts/publish-cloud-run.sh"
+      - "$_DIRECTORY"
+      - "$PROJECT_ID"
+      - "$_BUCKET"
+      - "$_KEY_LOCATION"
+      - "$_KEY_RING"
+      - "$_FUNCTION_REGION"
+      # override the bot name
+      - canary-bot-cloud-run
```

Comment:
I actually want to deploy the bot with the same name for my loadtest.
Maybe I can create a new bot for loadtesting.

Reply:
For posterity: we discussed adding an option to the publish cloud run script for overriding the queue name.